cmd/version/cmd.go

```diff
@@ -61,7 +61,7 @@ func Cmd() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "version",
-		Short: "Show the " + version.AppName + " version information",
+		Short: "📋 Show " + version.AppName + " version information",
```

Comment:
Oh I've moved this file as part of this PR:

https://github.com/datarobot-oss/cli/pull/155/files